composites/Plugin/Shared/components/HelpCenterButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import colors from "../../../../style-guide/colors.json";
+import { questionCircle, angleUp, angleDown } from "../../../../style-guide/svg";
+import { YoastButton } from "./YoastButton";
+import { Icon } from "../../Shared/components/Icon";
+
+/**
+ * Returns an icon button that can optionally contain text.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Styled icon button.
+ */
+const HelpCenterButtonBase = ( props ) => {
+	return (
+		<YoastButton { ...props }>
+			<Icon icon={ questionCircle } color={ props.iconColor } />
+			{ props.children }
+			<Icon icon={ props.isExpanded ? angleUp : angleDown } color={ props.iconColor } />
+		</YoastButton>
+	);
+};
+
+export const HelpCenterButton = styled( HelpCenterButtonBase )`
+	min-width: 0;
+	padding: 0 2px;
+
```

Comment:
newline needed?

Reply:
In plain CSS, `svg { ...` would be a new selector and yes, it would need a new line to be separated from the previous rule. In styled components, it's just a matter of readability I guess, and internal conventions.